spyder/utils/ipython/start_kernel.py

```diff
@@ -69,10 +69,12 @@ def kernel_config():
     # Until we implement Issue 1052
     spy_cfg.InteractiveShell.xmode = 'Plain'
 
+    # Jedi completer in IPpthon console

```

Comment:
Remove here `in IPpthon console`.

Reply:
This is still missing. Please address it.

Reply:
done